config/application.rb

```diff
@@ -13,10 +13,11 @@
 
 module PrisonVisits
   class Application < Rails::Application
+    config.active_record.schema_format = :sql
```

Comment:
I always end up here. Perhaps this should be the default schema format in Rails.